packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/WorkspaceSettings.tsx

```diff
@@ -76,11 +82,50 @@ export const WorkspaceSettings = () => {
   const [inviteValue, setInviteValue] = useState('');
   const [inviteLoading, setInviteLoading] = useState(false);
 
+  const [newMemberAuthorization, setNewMemberAuthorization] = React.useState<
+    TeamMemberAuthorization
+  >(
+    // get TeamMemberAuthorization key by value
+    Object.keys(TeamMemberAuthorization).find(key => {
```

Comment:
Mind moving this outside of the useState?

Mostly to look cleaner, no actual reason

Reply:
Simplified the types, so now it's not needed :)